ipmiMgrApp/Db/fru_common_alias.template

```diff
@@ -21,3 +21,28 @@ alias("$(dev):$(id)$(unit):PSN", "$(P)$(R)$(id)ProdSN-Cte")
 alias("$(dev):$(id)$(unit):PPARTNUMBER", "$(P)$(R)$(id)ProdPN-Cte")
 
 alias("$(dev):$(id)$(unit):POWERCTL", "$(P)$(R)$(id)PowerCtl-Sel")
+
+record(sseq, "$(P)$(R)$(id)SoftRst-Cmd") {
+  field(DESC, "Power cycle system")
+  field(SELM, "All")
+  field(PINI, "NO")
+  field(LNK1, "$(P)$(R)$(id)FruSts-Mon PP")
+  field(DO1, "1")
+  field(WAIT1, "Wait")
+  field(LNK2, "$(P)$(R)$(id)PowerCtl-Sel PP")
+  field(DO2, "0")
+  field(WAIT2, "Wait")
+  field(DLY3, "15")
+  field(LNK3, "$(P)$(R)$(id)PowerCtl-Sel PP")
+  field(DO3, "1")
+  field(WAIT3, "Wait")
+  field(DLY4, "15")
+  field(LNK4, "$(P)$(R)$(id)FruSts-Mon PP")
+  field(DO4, "0")
+  field(WAIT4, "Wait")
+}
+record(bo, "$(P)$(R)$(id)FruSts-Mon") {
+  field(ZNAM, "FRU_ON")
+  field(ONAM, "FRU_OFF")
```

Comment:
These names can be human friendly AFAIK. Also they are already associated with `FruSts`, specifying `FRU_` once again seems redundant for me. I'd rather simply call them `On` and `Off`.

I'm also a little bit confused by the meaning of the monitor value... does it mean "we are resetting now" (since we write 1 to it when `SoftRst-Cmd` is processing) or "the unit is not powered now" (since the `ONAM` is `Off`)?

IMHO, in case it is the first one, I'd use something more descriptive for its named values, e.g. `Restarting` and `Done`.

Reply:
You are right about the redundancy, maybe not using the FRU prefix is a better choice.
About the meaning of the monitor value, the idea is that it tracks the state of each unit on MCH. So, On and Off really mean that the FRU is on or off, and for this reason it starts in 0 (On), stays in 1 (Off) during all the reset process, and them turns back to one when the reset is done. Maybe, for disambiguation, I can call the 1 state `Restarting`. WDYT?